precompiles/pallet-xcm/XcmInterface.sol

```diff
@@ -29,68 +29,102 @@ interface XCM {
         uint256 amount;
     }
 
+    // The values start at `0` and are represented as `uint8`
+    enum TransferType {
+        Teleport,
+        LocalReserve,
+        DestinationReserve,
+        RemoteReserve
```

Comment:
```suggestion

```

Reply:
Done in aea402dd3193edd09242b0be19e07e3da4e07200.